less/<CardMatchHeight>.less

```diff
@@ -0,0 +1,14 @@
+.card-pf-match-height {
```

Comment:
Is there a reason to have the angle brackets in the filename?

Reply:
<img width="194" alt="screenshot 2018-02-26 16 21 19" src="https://user-images.githubusercontent.com/24938324/36675281-2e892d64-1b11-11e8-94c6-3e957f7896d1.png">

I was confused by this..fixing this shortly